x-pack/plugins/security_solution/public/management/cypress/tasks/isolate.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ActionDetails } from '../../../../common/endpoint/types';
+
+export const interceptActionRequests = (
```

Comment:
can you please add return types to all of these functions

Reply:
All these functions have no returns. I tend to separate cy selectors/actions that are being used in multiple places.

Reply:
No, these have returns, even if it's a `void`. Yes, Typescript will infer the return types, but having a explicit return type is best as it not only clarifies to the developer what the function is suppose to return, but also help to ensure that as changes occur inside of a given function, if those break the expected return type, then you get feedback immediately. 

IMO, I believe its best to ensure that functions (especially those that are `export`'d out) to be property typed, but if you feel strongly it is not necessary, then disregard (although, I will continue to provide this type of feedback in code reviews 😄 ).

Reply:
Since you have a strong opinion on the matter and I don't, I will add these explicit returns :)